README.md

```diff
@@ -1 +1,78 @@
-# Hello-World
\ No newline at end of file
+# Hello-Microverse
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
```

Comment:
- [ ] You did a good job working on setting up your first project at Microverse! 💯 However,  

- [ ] Please kindly note that all main headers from the [README template](https://github.com/microverseinc/readme-template) should be present in your repo and their contents are customized accordingly. The only optional section is FAQ.

### TEMPLATE HEADERS 👇 
![image](https://user-images.githubusercontent.com/78251927/226734172-33758545-2d06-4fa4-a5a4-791a53cb9ef2.png)

- [ ] A README file is a guide that gives users a detailed description of a project you have worked on. so it is always necessary to have a descriptive README.